subprojects/core/src/integTest/groovy/org/gradle/api/tasks/bundling/ConcurrentArchiveIntegrationTest.groovy

```diff
@@ -437,22 +438,14 @@ class ConcurrentArchiveIntegrationTest extends AbstractIntegrationSpec {
         given: "2 archive files"
         createTar('test1.tar') {
             subdir1 {
-                file ('file.txt').text = 'original text 1'
+                file('file.txt').text = 'original text 1'
             }
             subdir2 {
                 file('file2.txt').text = 'original text 2'
-                file ('file3.txt').text =  'original text 3'
-            }
-        }
-        createTar('test2.tar') {
-            subdir1 {
-                file ('file.txt').text = 'original text 1'
-            }
-            subdir2 {
-                file('file2.txt').text = 'original text 2'
-                file ('file3.txt').text =  'original text 3'
+                file('file3.txt').text = 'original text 3'
             }
         }
```

Comment:
:grey_question:  This is surprising! How come if you copy it (and not recreate it) solves this?

Reply:
If you don't do something about it, archiving files includes their modification/creation timestamps too, which might differ, even though you create them in rapid succession.

See for example [our AbstractArchiveTask](https://github.com/gradle/gradle/blob/80479d0f23dc210306fede37d89836a569807ca7/subprojects/core/src/main/java/org/gradle/api/tasks/bundling/AbstractArchiveTask.java#L273).